projects/js-packages/components/components/pricing-slider/style.scss

```diff
@@ -0,0 +1,55 @@
+@import '@automattic/jetpack-base-styles/style';
+
+$thumb-height: 40px;
+$track-height: 8px;
+
+@mixin adjust-track-rail-styles {
+	height: $track-height;
+	border-radius: 8px; /* stylelint-disable-line scales/radii */
+}
+
+// Base styles
+.pricing-slider {
+	width: 100%;
+    height: $thumb-height;
+
+    .pricing-slider__track {
+        @include adjust-track-rail-styles;
+        top: calc(($thumb-height / 2) - ($track-height / 2));
```

Comment:
I think we need to fix the indentation style in this file, which uses both spaces and tabs.

Reply:
Fixed in [2c43d33](https://github.com/Automattic/jetpack/pull/31425/commits/2c43d3362265d02461b4129abf4f87f093c47df9).